app/models/pawn.rb

```diff
@@ -7,4 +7,54 @@ def unicode_symbol
       return "&#9823;"
     end
   end
+
+  def valid_move?(destination_x, destination_y)
+    valid = super(destination_x, destination_y)
+    if valid 
+      if pawn_diagonal?(destination_x, destination_y)
+        capture_piece(destination_x, destination_y)
+        return true
+      elsif move_backward?(destination_y) && (one_square?(destination_x, destination_y) || two_squares?(destination_x, destination_y))
+        return false
+      end
+      move_backward?(destination_y) 
+    end
+    return valid
+  end
+
+  def on_home_row?
+    return true if (self.get_color == WHITE && y_position == 1) || y_position == 6
+  end
+
+  def one_square?(destination_x, destination_y)
+    return !on_home_row? && ((destination_x – self.x_position).abs > 0 || (destination_y – self.y_position).abs > 1)
+  end
+
+  def two_squares?(destination_x, destination_y)
+    return on_home_row? && ((destination_x - self.x_position).abs > 0 || (destination_y - self.y_position).abs > 2)
+  end
+
+  def pawn_diagonal?(destination_x, destination_y)
+    if game.is_piece_present?(destination_x, destination_y) 
```

Comment:
@kellylougheed - you mean different color for capturing, right? I get what you're saying. I added `&& self.get_color == other_piece.get_color` after `if game.is_piece...` to make sure the color is checked in that statement. Is this what you mean?